orttraining/orttraining/core/framework/gradient_graph_builder.h

```diff
@@ -61,7 +61,10 @@ static std::unordered_map<std::string, std::unordered_set<size_t>>
         {"Range", {0, 1, 2}},
         {"Tile", {1}},
         {"BroadcastGradientArgs", {0, 1}},
-        {"TopK", {1}}};
+        {"TopK", {1}},
```

Comment:
Another addition should be "Split", {1} as the opset 13 version of Split has an extra input 'split'